src/app.js

```diff
@@ -170,6 +231,38 @@ app.on('ready', () => {
     }
   }
 
+  function setNotificationTime(event, time) {
+    notificationWindow.close();
+    if (time && time !== store.get('notification.time')) {
+      const hours = time.split(':')[0];
+      const minutes = time.split(':')[1];
+      store.set('notification.time', time);
+      store.set('notification.hours', hours);
+      store.set('notification.minutes', minutes);
+      log.info(`Store updated - time=${time} hours=${hours} minutes=${minutes}`);
+      job.setTime(new CronTime(`0 ${minutes} ${hours} * * *`));
+      job.start();
+    }
+  }
+
+  const job = new CronJob({
+    cronTime: `0 0 20 * * *`,
+    async onTick() {
+      const data = await contribution(store.get('username'));
+      if (data.currentStreak === 0 && Notification.isSupported()) {
+        new Notification({
+          title: 'Streaker',
```

Comment:
I don't forsee the project name ever changing however, I've avoided hardcoding the name anywhere in the app in favour of getting it from the package json, i.e. `pjson.name`